tests/unit/lib/bootstrap/test_bootstrap.py

```diff
@@ -13,7 +13,7 @@
 
 class TestBootstrapManagedStack(TestCase):
     def _stubbed_cf_client(self):
-        cf = botocore.session.get_session().create_client("cloudformation")
+        cf = botocore.session.get_session().create_client("cloudformation", region_name="us-west-2")
```

Comment:
why hardcode?

Reply:
This is a unit tests and the tests will fail unless you have defined a region or have it on the system as an env var.